### ISSUE TITLE
Microsoft.WindowsAzure.Storage.StorageException: 'The specified queue does not exist.'

### ISSUE THREAD

**Issue by symlynk** [Azure/azure-webjobs-sdk#1483]
I am seeing the following exception continuously being thrown and displayed in the output window:

Exception thrown: 'Microsoft.WindowsAzure.Storage.StorageException' in System.Private.CoreLib.dll

When running this code as a .NET Core 2.0 console app with no other classes apart from the following:

```
    class Program
    {
        static void Main( string[] args )
        {
            var config = new JobHostConfiguration();

            config.DashboardConnectionString = "..."; // replace ... with storage connection string
            config.StorageConnectionString = "..."; // replace ... with storage connection string

            var host = new JobHost( config );

            host.RunAndBlock();
        }
    }
```

The exception is being thrown from StorageQueue.cs with the message:

> "Microsoft.WindowsAzure.Storage.StorageException: 'The specified queue does not exist.'"

Microsoft.Azure.WebJobs (3.0.0-beta4)

**Comment by erick-thompson** [comment]
I'm running into this issue as well. Any update on what it might be?

**Comment by brettsam** [comment]
Are you able to catch and inspect the queue name? WebJobs attempts to monitor a 'host' queue for requests to shutodown -- but that queue doesn't always exist. It's likely this is just a background exception that we're already handling, but if you can track down the queue name, that would confirm it.

**Comment by brettsam** [comment]
FYI -- here's a related issue: https://github.com/Azure/azure-webjobs-sdk/issues/1215

**Comment by erick-thompson** [comment]
I am able to catch the StorageException, but I can't figure out how to get the queue name from the exception. I've poked around inside the StorageException, but no luck. I have the ServiceRequestId, but nothing else that looks promising. Any suggestions on how to get the queue name?

**Comment by erick-thompson** [comment]
I just fired up Fiddler. It appears to be hitting a queue named "azure-webjobs-host-<machinename>".

**Comment by erick-thompson** [comment]
After digging into the code, I see where and how it is used. However, I can't find any documentation for it. Is this queue to be used by the SDK only, or is it something that can be used by developers?

**Comment by brettsam** [comment]
See #770 for an explanation of what those queues are for.

**Comment by mathewc** [comment]
Closing this issue since https://github.com/Azure/azure-webjobs-sdk/pull/1572 addresses it. https://github.com/Azure/azure-webjobs-sdk/issues/1215 remains open to track potential improvements to AppInsights to correctly handle non-error logs.

**Comment by ggirard07** [comment]
same issue here while running on the storage emulator. But that behavior is happening only on my machine, not on the ones of my coworker.

The name of the queues involved are:
- `azure-webjobs-host-[heartbeatsId]` where `heartbeatsId` is a Guid I found in `http://127.0.0.1:10000/devstoreaccount1/azure-webjobs-hosts/heartbeats/`
- `azure-webjobs-host-[heartbeatsBlobId]` where `heartbeatsBlobId` is the Guid of the latest blob in `http://127.0.0.1:10000/devstoreaccount1/azure-webjobs-hosts/heartbeats/09cadd1626424a3b8c52a7f0e985dfd0/`

Is #1572 going to address my case too (no AppInsight here)?
That fix is still not included in v2.2.0 release, right?
